build.sh

```diff
@@ -261,6 +267,34 @@ function build_sysrel {
     echo "TODO!!"
 }
 
+function build_frontend {
+
+    # Making sure repo was cloned (needed even if images are already built)
+    if [ ! -d "dlr-riesgos-frontend" ]; then
+        git clone https://github.com/riesgos/dlr-riesgos-frontend --branch=compare-frontend # --branch=2.0.6-main <-- once we have a stable tag
+    fi
+
+    # Checking if rebuilding is required
+    buildIt=false
+    frontendImages=("dlr-riesgos-frontend-frontend" "dlr-riesgos-frontend-compare-frontend" "dlr-riesgos-frontend-monitor" "dlr-riesgos-frontend-backend")
+    for image in "${frontendImages[@]}"
+    do
+        if misses_image "$image"; then
+            buildIt=true
+        fi
+    done
+
+    # Build if required
+    if [ "$buildIt" = false ]; then
+        echo "Already exists: frontend"
+    else
+        cp .env ./dlr-riesgos-frontend/
+        cd ./dlr-riesgos-frontend
+        docker compose build
+        cd SCRIPT_DIR
```

Comment:
No particular reason - just seamed safer. Sometimes we step several dirs deep into a cloned repo - I figured it would be useful to have a reference to the base-dir.
Can remove it, though!

Reply:
No it is ok, I think.
But then we should set it in all the other build steps - just to improve the overall script & be save on that point.